src/scripts/main.js

```diff
@@ -1,13 +1,30 @@
 'use strict';
 
+const form = document.querySelector('form');
+
 function waitFor(element, eventName) {
-  // write your code here
+  return new Promise((resolve, reject) => {
+    if (element) {
+      element.addEventListener(eventName, () => {
+        const string = `It was ${eventName} on the element: ${element.nodeName}, id: ${element.id}.`;
+
+        resolve(string);
+      });
+    } else{
+      reject(new Error('Element doesn\'t exist'));
+    }
+  });
 }
 
+const messageDiv = document.createElement('div');
+messageDiv.classList.add('message');
+messageDiv.style.display = 'none';
 const printMessage = (message) => {
```

Comment:
This function changes the visibility of the messageDiv and changes its text. It would be better to divide it into two separate functions: one for changing visibility and one for changing the text.